windows/devices/uwb/UwbCxAdapter.cxx

```diff
@@ -0,0 +1,239 @@
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](uint8_t arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](uint16_t arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint16_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](uint32_t arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint32_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](::uwb::protocol::fira::Channel arg) {
+                           uint8_t data = 9;
+                           switch (arg) {
+                           case (::uwb::protocol::fira::Channel::C5): {
+                               data = 5;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C6): {
+                               data = 6;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C8): {
+                               data = 8;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C9): {
+                               data = 9;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C10): {
+                               data = 10;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C12): {
+                               data = 12;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C13): {
+                               data = 13;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C14): {
+                               data = 14;
+                               break;
+                           }
+                           }
+                           builder.Push(std::move(UwbAppConfigurationParameter{ data, service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](::uwb::protocol::fira::ConvolutionalCodeConstraintLength arg) {
+                           // TODO
+                       },
+                       [&builder, ptag](::uwb::protocol::fira::DeviceRole arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
```

Comment:
All of the `enum class`'es that match parameter tags have been defined such that their underlying type is `uint8_t`. If there is concern about this possibly mismatching, then you can add a `static_assert` to guarantee it, which is never a bad idea (I might do that someplace else, maybe at the very beginning of this function since it's an assumption this function relies on). So, I think `notstd::to_underlying` should be used here (and other similar instances) instead. And if not, at least a `static_cast<uint8_t>` should be preferred over a C-style cast.